src/helpers/edit.js

```diff
@@ -1,5 +1,18 @@
 import { findIndex, map } from 'lodash'
 
+function isProjectStaffed(projectRow) {
+  const projectName = projectRow.project
+  const isStaffed = Object.keys(projectRow.staffing).reduce((acc, val) => {
+    if (acc) return acc
+    const projectStaffing = projectRow.staffing[val][projectName]
+    if (projectStaffing !== null && projectStaffing > 0) {
+      return true
+    }
+    return false
+  }, false)
```

Comment:
hmmm unsure if this is readable enough

Reply:
Will rename some variables to make it a bit more clear

Reply:
Swapped to use .some() instead of .reduce()